### PR TITLE
[9.0][NEW][WIP] product_supplierinfo_revision: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
Module to duplicate supplierinfo items with variation percentage price and new start date in new item, and set end date as previos day.

@Tecnativa

### CODE REVIEW

**Review by rigo1985 — APPROVED** [review]
Tested in runbot. Good contribution

File: product_supplierinfo_revision/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+This module allow create revisions of supplier info prices.
+
+
+Usage
+=====
+
+To use this module:
+
+#. Go to ...
```
Comment (yajo) [inline product_supplierinfo_revision/README.rst]: Please include proper instructions so we know what to test.

File: product_supplierinfo_revision/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Supplierinfo Revision",
+    "summary": "Customizations for Licores Reyes",
+    "version": "9.0.1.0.0",
+    "category": "Product",
+    "website": "http://www.tecnativa.com",
+    "author": "Tecnativa S.L.",
```
Comment (yajo) [inline product_supplierinfo_revision/__openerp__.py]: This must be "Tecnativa,Odoo Community Association (OCA)".

File: product_supplierinfo_revision/views/product_supplierinfo_view.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                <!--<div>-->
+                    <!--<field name="variation_percent" class="oe_inline"/>-->
+                <!--</div>-->
+                <!--<label for="previous_price"/>-->
+                <!--<div>-->
+                    <!--<field name="previous_price" class="oe_inline"/>-->
+                <!--</div>-->
+            <!--</xpath>-->
+        <!--</field>-->
+    <!--</record>-->
```
Comment (yajo) [inline product_supplierinfo_revision/views/product_supplierinfo_view.xml]: Delete this please.

**Review by yajo — CHANGES_REQUESTED** [review]
Please add tests.

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Core review and tested.

**Review by lasley — APPROVED** [review]
Code review

**Pull request merged**